packages/xo-server/src/xo-mixins/pool.js

```diff
@@ -81,4 +81,35 @@ export default class Pools {
       await _xo.mergeXenPools(source._xapiId, target._xapiId, force)
     }
   }
+
+  async listMatchingCriteria({ cpusCount = 0, memorySize = 0, poolTag, srSize = 0, srTag }) {
```

Comment:
Better add a **min**  or **exceeds** prefix to make it clear that it will not match the exact passed value.

example:

**cpusCount** -> **minCores**?

Reply:
I agree: `minCpus`, `minMemory`, `minSrSize`, etc.